mava/environment_loop.py

```diff
@@ -631,14 +614,12 @@ def should_run_loop(eval_interval_condition: Tuple) -> bool:
                     # Check for extra logs
                     if hasattr(self._environment, "get_interval_stats"):
                         results.update(self._environment.get_interval_stats())
-                    self._logger.write(results)
```

Comment:
Please keep this at this level of indentation.

Reply:
isn't working fine if we leave it out the if statement since it is called in line 641 too?

Reply:
Oh I see I saw the difference